crates/rattler_solve/tests/snapshots/backends__resolvo__solve_dummy_repo_optional_depends_recursive_feature.snap

```diff
@@ -0,0 +1,13 @@
+---
+source: crates/rattler_solve/tests/backends.rs
+expression: result.unwrap_err()
+---
+Cannot solve the request because of: The following packages are incompatible
+└─ foo[with-recursive] * cannot be installed because there are no viable options:
+   └─ foo[with-recursive] 2.0.2 would require
+      ├─ foo ==2.0.2 py36h1af98f8_1[md5=d65ab674acf3b7294ebacaec05fc5b54, sha256=1154fceeb5c4ee9bb97d245713ac21eb1910237c724d2b7103747215663273c2], which can be installed with any of the following options:
+      │  └─ foo 2.0.2
+      └─ foo[with-bar] *, which cannot be installed because there are no viable options:
```

Comment:
This error message is also kinda vague.. If I read it, it states that `foo[with-recursive] *` requires, `foo[with-bar]`??